HLTrigger/Timer/plugins/FastTimerServiceClient.cc

```diff
@@ -418,8 +418,8 @@ FastTimerServiceClient::fillPlotsVsLumi(DQMStore::IBooker & booker, DQMStore::IG
     label.erase(label.find("_byls"));
 
     MonitorElement* me = getter.get(current_path + "/" + m);
-    double ymin        = me->getTProfile()->GetMinimum();
-    double ymax        = me->getTProfile()->GetMaximum();
+    float ymin        = 0.;
+    float ymax        = std::numeric_limits<float>::max();
```

Comment:
hi @mtosi - naively it looks to be better just to drop the ymin and ymax arguments from the bookProfile call below. Why keep them?